ManagedCache+CoreDataClass.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  ManagedCache+CoreDataClass.swift
+//  FeedStoreChallenge
+//
+//  Created by Mher Movsisyan on 28.08.21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+//
+
+import Foundation
+import CoreData
+
+@objc(ManagedCache)
+class ManagedCache: NSManagedObject {
```

Comment:
This class is not intended to be inherited thus it should be marked as `final`.
```suggestion
final class ManagedCache: NSManagedObject {
```